manifests/config.pp

```diff
@@ -98,7 +98,7 @@
   }
 
   file { $dnscache::root_servers:
-    source => $dnscache::root_servers_source,
+    content => join($dnscache::root_server_ips, "\n"),
```

Comment:
Yes. `join` is part of stdlib which is already a dependency for this module. see https://github.com/puppetlabs/puppetlabs-stdlib/tree/3.0.0#join. It should work on 2.7, I could write a simple test for it and travis can autocheck the stuff if you want.